test/Liip/RMT/Tests/Functional/GitTest.php

```diff
@@ -76,6 +76,6 @@ public function testTagPrefixWithBranchNamePlaceHolder(): void
         exec('./RMT release -n --confirm-first');
         exec('git tag', $tags);
 //        $this->manualDebug();
-        self::assertEquals(array('_main_1'), $tags);
+        self::assertEquals(['_main_1'], $tags);
```

Comment:
sorry for the noise. now i understand whats going on.

```suggestion
        self::assertEquals([sprintf('_%s_1', self::DEFAULT_FUNCTIONAL_BRANCH)], $tags);
```